apps/aehttp/priv/oas3.yaml

```diff
@@ -1080,7 +1078,12 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/EncodedTx"
+              type: object
```

Comment:
Is this not "breaking" the specification in such a way that you would need a new API version number??
You re-use the same name, but give it a different meaning.